tests/test_database.py

```diff
@@ -4206,6 +4210,12 @@ def test_long_names_dbc(self):
         self.assertEqual(db.messages[7].signals[2].receivers,
                          ['N123456789012345678901234567890123'])
 
+        # environment variables
+        envvar_names = db.dbc.environment_variables
+        self.assertTrue('E1234567890123456789012345678901' in envvar_names)
```

Comment:
Use `assertIn()` and `assertNotIn()` instead.

Reply:
`assertIn()` and `assertNotIn()` unfortunately are not available in python 2.x. So I've used the more general `assertTrue()` to get a higher test coverage.
Are there further reasons to prefer the more specific methods?

Reply:
Ohh, ok. Nevermind then.

Reply:
I just had a look at [docs.python.org](https://docs.python.org/3/library/unittest.html?highlight=assertin#assert-methods). They say "new in 3.1"

Reply:
https://docs.python.org/2.7/library/unittest.html?highlight=assertin#assert-methods

Reply:
Hmm, that's funny. I'll adapt the code and hope, that your link wins about mine ;-)

Reply:
=)